ops/ansible/roles/common/tasks/bootstrap.yaml

```diff
@@ -0,0 +1,36 @@
+- name: Check ubuntu user
+  command: ssh -q -o BatchMode=yes -o ConnectTimeout=3 ubuntu@{{ inventory_hostname }} -p {{ ansible_ssh_port }} "echo OK"
+  delegate_to: 127.0.0.1
+  changed_when: false
+  failed_when: false
+  register: check_ubuntu_user
+  tags: bootstrap
+
+- block:
+  - name: Add the user 'ubuntu' with a specific groups
+    ansible.builtin.user:
+      name: ubuntu
+      groups: sudo
+      password: "$1$IveBeenA$UlFfzeZc0GDrihfkKvKdY."
```

Comment:
Not knowing much about ansible... I presume the $ signs are some sort of interpolation?

Reply:
as discussed - they are encoding of the salt used to encrypt this